terrautils/sensors.py

```diff
@@ -118,75 +121,94 @@
               "fixed_metadata_datasetid": "TBD"
         },
 
-        'rgb_fullfield': {
+        'fullfield': {
+            'display': 'Full Field Stitched Mosaics',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{date}{opts}.tif',
         },
 
-        'flirIrCamera': {
+        'hyperspectral2nc_vnir': {
+            'display': 'VNIR Hyperspectral NetCDFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
-            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
+            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
+        },
+
+        'hyperspectral2nc_swir': {
+            'display': 'SWIR Hyperspectral NetCDFs',
+            'template': '{base}/{station}/Level_1/'
+                        '{sensor}/{date}/{time}/{filename}',
+            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
         },
 
         'stereoTop_geotiff': {
```

Comment:
In practice this is just an RGB camera, and it will be easier to understand that these are RGB geotiffs if RGB were included in the name like `stereoTop_RGB_geotiff`. (We could drop `stereoTop` but then it wouldn't be clear that which sensor was used.)